.github/ISSUE_TEMPLATE/notify-dependency-update-template.md

```diff
@@ -1,11 +1,3 @@
----
-name: Notify Dependency Update Template
-about: Regular dependency updates
-title: Regular Update for Dependencies
-labels: Notify, QA, Tech Debt
-assignees: ''
----
-
```

Comment:
The Github API acts up when using `--template` to create issues in a non-interactive shell. In order to get this to work, I had to use `--body-file` and remove this metadata from the top of the file. I figured this was fine, since we plan on creating these automatically now.